addons/mail/static/src/components/discuss/tests/discuss_sidebar_tests.js

```diff
@@ -138,6 +138,26 @@ QUnit.test('sidebar find shows channels matching search term even when user is m
     );
 });
 
+QUnit.test('sidebar channels should ordered alphabetically', async function (assert) {
+    assert.expect(2);
+
+    this.data['mail.channel'].records.push({ id: 19, name: "Xyz" }, { id: 20, name: "abc" },
+                                        { id: 21, name: "Abc" }, { id: 22, name: "Xyz" });
```

Comment:
Guideline is either single line or one element per line, not a mix. In this case since there are 4 of them, I would expect to see 4 lines.